### PR TITLE
Release Godot 4.2 beta 6 and publish release notes

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-2-beta-6.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- GUI/Input: Make mouse enter/exit notifications match mouse events ([GH-84547](https://github.com/godotengine/godot/pull/84547)).
+  * This should restore the expected behavior for `mouse_entered` and `mouse_exited` signals. If you had noticed any weird change in mouse events in your projects in earlier betas, please make sure to test this beta 6 and confirm if your bugs are properly fixed.
+- Input: Rework input actions to be reliable ([GH-84685](https://github.com/godotengine/godot/pull/84685)).
+  * Godot 4.2 fixes an infamous bug affecting actions with multiple input methods assigned to them ([GH-45628](https://github.com/godotengine/godot/issues/45628)), but that fix relied on getting properly information from the OS on which key events are repeats (echo). Turns some OSes ([GH-82262](https://github.com/godotengine/godot/issues/82262)) and browsers ([GH-82732](https://github.com/godotengine/godot/issues/82732)) don't properly report this information, so we had to take another stab at refactoring this input handling. We need it tested thoroughly, especially on projects supporting using multiple event types at the same time.
```
Comment (YuriSizov) [inline collections/_article/dev-snapshot-godot-4-2-beta-6.md]:
```suggestion
  * Godot 4.2 fixes an infamous bug affecting actions with multiple input methods assigned to them ([GH-45628](https://github.com/godotengine/godot/issues/45628)), but that fix relied on getting proper information from the OS about which key events are repeated (echo). Turns out some OSes ([GH-82262](https://github.com/godotengine/godot/issues/82262)) and browsers ([GH-82732](https://github.com/godotengine/godot/issues/82732)) don't properly report this information, so we had to take another stab at refactoring this input handling. We need it tested thoroughly, especially on projects supporting using multiple event types at the same time.
```

File: collections/_article/dev-snapshot-godot-4-2-beta-6.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+- To export to Android, .NET 7.0 or later is required. To export to iOS, .NET 8.0 is required. Make sure to set the target framework in the `.csproj` file.
+
+{% include articles/prerelease_notice.html %}
+
+## Known issues
+
+We're currently tracking issues that we consider release blockers very closely. The main remaining ones that we're aware of are below:
+
+- Following [GH-84657](https://github.com/godotengine/godot/pull/84657), the .NET-enabled Godot editor seems to get stuck on exit ([GH-84728](https://github.com/godotengine/godot/issues/84728)). We're working on solving this for the next build.
+- If you use both Godot 4.1 and 4.2 as self-contained with separate caches, you may run into this edge case: [GH-84732](https://github.com/godotengine/godot/issues/84732). Deleting the `.godot` folder and doing a clean import with Godot 4.2 should work it around.
```
Comment (YuriSizov) [inline collections/_article/dev-snapshot-godot-4-2-beta-6.md]:
```suggestion
- If you use both Godot 4.1 and 4.2 as self-contained with separate caches, you may run into this edge case: [GH-84732](https://github.com/godotengine/godot/issues/84732). Deleting the `.godot` folder and doing a clean import with Godot 4.2 should work around this problem.
```

Work it, baby? 🙃

**Review by YuriSizov — APPROVED** [review]
Two corrections, should be good otherwise.

**Comment by JohnVeness** [comment]
I'm not sure what you mean by "relatively core" in the excerpt and second paragraph?

File: collections/_article/dev-snapshot-godot-4-2-beta-6.md (hunk 1 above)
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-2-beta-6.md]:
```suggestion
  * Godot 4.2 fixes an infamous bug affecting actions with multiple input methods assigned to them ([GH-45628](https://github.com/godotengine/godot/issues/45628)), but that fix relied on getting proper information from the OS on which key events are repeats (echo). It turns out that some OSes ([GH-82262](https://github.com/godotengine/godot/issues/82262)) and browsers ([GH-82732](https://github.com/godotengine/godot/issues/82732)) don't properly report this information, so we had to take another stab at refactoring this input handling. We need it tested thoroughly, especially on projects supporting using multiple event types at the same time.
```

**Comment by YuriSizov** [comment]
> I'm not sure what you mean by "relatively core" in the excerpt and second paragraph?

It means that this is a critical change, as it's pretty close to the core of the engine and can affect everything.

**Review by Calinou — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks for the reviews!

**Pull request merged**